### DISCUSSION TITLE
About XPU_TIMER.

### DISCUSSION THREAD

**Discussion by BalaBalaYi** [intelligent-machine-learning/dlrover#1350] [Announcements]
We sincerely apologize, the previously planned release date has been delayed.
Current planned release date (not guaranteed, preparing for academic related work for now): Q1 2025

For further information and discussion can be done in this issue.

**Comment by aqwertaqwert** [comment]
Does xpu_time have plans to support Huawei npu cards?

**Reply by BalaBalaYi** [reply]
> Does xpu_time have plans to support Huawei npu cards?

In fact, this is not a technical issue. This depends on Huawei's stance. U know what i mean.

**Reply by aqwertaqwert** [reply]
So we already support this feature, but cannot open source it?

**Comment by liu-yx17** [comment]
For the xpu_timer experience process, in the Chinese mainland, building WHL via Bazelist is prone to problems, because the server are most likely not able to access the foreign website to get the required compressed package. Can a xpu_timer docker image be provided for use?

**Reply by BalaBalaYi** [reply]
image is provided

**Comment by dbfancier** [comment]
Hi, after building the xpu_timer, the relevant lib could not be found, and needed to be packaged and installed to /usr/local/lib and /usr/local/bin. The generated matmul_dp8.bin after all this was done seemed to be problematic.

Using xpu_timer_gen_tracing_timeline also seems to produce svg that does not properly reflect timeline information. Are there more documentation or communication group to discuss these issues?

**Reply by BalaBalaYi** [reply]
replied in issue

**Comment by BalaBalaYi** [comment]
XPU Timer's 1st edition already be released.
